### DISCUSSION TITLE
iOS 기기끼리 직접적으로 데이터/신호를 주고받는 방법엔 무엇이 있을까요? (+NFC 이야기)

### DISCUSSION THREAD

**Discussion by yeniful** [DeveloperAcademy-POSTECH/.github#39] [Topic]
안녕하세요 1기 예니입니다.
최근 iOS 기기끼리 직접적으로 NFC로 데이터를 주고받을 수 있는지에 대한 이야기가 오갔었어요.
AOS에서는 잘 사용했던 기억이 있었기에 iOS에서는 어떤지 궁금해져서 저도 서치를 해보았는데요.
알아본 결과 iOS는 기기끼리 직접적으로 NFC로 데이터를 주고받는 것은 불가능한 것으로 파악됐어요.

NFC 리더기를 통해 NFC 태그에 데이터를 쓰거나 데이터를 읽는 원리인데, 아이폰은 NFC 리더기이고, NFC 태그 역할을 하지는 않는다고 하더라고요. 
애플페이만 예외적으로 NFC 카드 에뮬레이션 모드로 애플페이 리더기와 통신할 때 카드(태그)처럼 인식된다고 합니다.

**그렇다면 iOS 기기끼리 직접적으로 데이터/신호를 주고받을 때 어떤 기술이나 프레임워크들을 활용하면 좋을까요?
해당 방법이 어떤 특징과 장단점이 있는지도 함께 이야기 나누어보면 좋을 것 같아요 👀**

**Comment by insub4067** [comment]
큐알코드요! 장점 : 비교적 자유롭고 편한 구현 단점 : 데이터의 크기 제한
